othello/templates/emails/tournament_notify.txt

```diff
@@ -0,0 +1,17 @@
+Hello! This is an automated message from the Othello Tournament Server.
+
+You are receiving this email because your AI was selected to participate in an Othello Tournament
+If you receive this email multiple times this means the tournament has been rescheduled. The most recent email is always the most accurate
```

Comment:
```suggestion
You are receiving this email because your AI was selected to participate in an Othello tournament.
If you receive this email multiple times, the tournament has been rescheduled. The most recent email is always the most accurate.
```